tests/providers/amazon/aws/waiters/test_glue_databrew.py

```diff
@@ -66,3 +66,4 @@ def test_job_succeeded(self, mock_describe_job_runs):
         ]
         waiter = GlueDataBrewHook(aws_conn_id=None).get_waiter("job_complete")
         waiter.wait(name=self.JOB_NAME, runId=self.RUN_ID, WaiterConfig={"Delay": 0.2, "MaxAttempts": 2})
+        waiter.wait.assert_called()
```

Comment:
```suggestion
```

`waiter.wait` is not mocked, so assert_called method is not available.

Reply:
I'll remove it.